test-avocado/lib/guest-cockpit.sh

```diff
@@ -37,6 +37,9 @@ function yum_builddep() {
     yum-builddep -y "$@" | (grep -v -- "--> Already installed : \|Getting requirements for\|No uninstalled build requires" || true)
 }
 
+echolog "Updating OS"
+yum -y update
+
```

Comment:
I was hoping not, but this has completely wrecked checkmachine7 on the first run.  Even systemd didn't respond anymore.  Is it not supported to run "yum update" over ssh?  I do that all the time with my development VMs...